pom.xml

```diff
@@ -462,10 +468,22 @@
                 <artifactId>netty-handler</artifactId>
                 <version>${version.netty}</version>
             </dependency>
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty-buffer</artifactId>
+                <version>${version.netty}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.netty</groupId>
+                <artifactId>netty</artifactId>
+                <version>3.10.6.Final</version>
```

Comment:
Why are we using a different version for `io.netty.netty` than for all the other netty dependencies?

Also, assuming we do need a separate version, we should pull it out into a variable and use the variable rather than hard-coding it.

Reply:
This is for fixing a vulnerability check. If I remember correctly, the `netty` doesn't share the same versions with the other `netty-*`